components/LeagueCard/LeagueCard.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { cn } from '@/utils/utils';
+import Image from 'next/image';
+import placeholderImage from './placeholderImage.svg';
+
+interface ILeagueCardProps extends React.HTMLAttributes<HTMLDivElement> {
+  isEliminated?: boolean;
+}
+
+const LeagueCard = React.forwardRef<HTMLDivElement, ILeagueCardProps>(
+  ({ className, isEliminated = false, ...props }, ref) => (
+    <div
+      data-testid="LeagueCard"
+      ref={ref}
+      className={cn(
+        'flex h-32 place-items-center gap-6 rounded-lg border bg-card p-4 text-card-foreground shadow-sm dark:border-zinc-800',
+        className,
+        { 'LeagueCardEliminated opacity-50 dark:bg-zinc-700': isEliminated },
+      )}
+      {...props}
+    />
+  ),
+);
+LeagueCard.displayName = 'LeagueCard';
+
+const LeagueCardHeader = React.forwardRef<
```

Comment:
Every one of these components should be their own files. And each component needs its own independent unit tests.

Reply:
Separate files created. Unit tests to come.

Reply:
All separate unit tests in latest commit.